provider/pact-jvm-provider-maven/src/main/kotlin/au/com/dius/pact/provider/maven/PactProviderMojo.kt

```diff
@@ -136,8 +136,12 @@ open class PactProviderMojo : AbstractMojo() {
     val options = mutableMapOf<String, Any>()
 
     if (pactBroker?.authentication != null) {
-      options["authentication"] = listOf("basic", provider.pactBroker.authentication.username,
-        provider.pactBroker.authentication.password)
+      if (provider.pactBroker.authentication.password != null) {
```

Comment:
This logic should be based on the scheme. If someone sets the scheme to 'bearer', but then sets the password to a non-empty value, this will not work.

Better to have a token attribute, and we can default the scheme to 'bearer' if that is set.

Reply:
done, and added a unit test to assert the new behavior